COLLABORATOR_GUIDE.md

```diff
@@ -452,6 +452,15 @@ Check number of commits and commit messages
 $ git log upstream/master...master
 ```
 
+If you land a somebody else's PR (especially a first-time contributor's PR),
+please make sure that a username and an email in the log match the ones
+in the author's GitHub account. Otherwise, the commits will not be properly
+associated with the author's account and the first-time contributor
```

Comment:
should there be a comma after `author's account`?

Reply:
Done.